### ISSUE TITLE
ImagesAPI.RandomNsfw(bool gifs) seems to ignore the actual boolean

### ISSUE THREAD

**Issue by TazmamzaT** [KSoft-Si/KSoft.Net#24]
I did a bit of testing with this on my discord bot that wasn't working properly (and removed) [here](https://github.com/TazmamzaT/TazBot/commit/750756e8d9450d0faadc063aff70a8fd045e9883). I tried using the bool a bit and not using the bool a bit and I wasn't ever able to get an actual gif to load.

**Comment by TazmamzaT** [comment]
I found the issue I think.

https://github.com/KSoft-Si/KSoft.Net/blob/787f48974535b3ce7c2b96b478071547ce72fd01/KSoftNet/KSoft/ImagesAPI.cs#L74

Calling ToString on the boolean converts it to "True" and the api doesn't seem to recognize that. I tested with manually setting the flag in DoShitAsync to "true" and with calling ToLower in ImagesAPI and both worked. 

https://github.com/TazmamzaT/KSoft.Net/commit/b05a8f09a44ce73c3b145266d755ea3e44fcc4d2
